int32[] obj_ids
int32 desc_id
---
